src/migrations/simple_migration.js

```diff
@@ -0,0 +1,135 @@
+import Migration from './migration';
+
+export default class SimpleMigration extends Migration {
+  constructor(configuration) {
+    super(configuration);
+    this._defaultIndex = configuration.config.get('kibana.index');
+    this._defaultType = 'doc';
+
+  }
+
+  /**
+   * @return {number}
+   */
+  get newVersion() {
+    if (!this._newVersion) {
+      throw new Error('Every SimpleMigration must declare a new object version (positive integer).');
+    }
+    return parseInt(this._newVersion);
```

Comment:
How do you think we can add a measure to prevent people from mistakenly using a lesser version than the previous migration for the same object?
I think it might require some changes in the way we register migrations.

Reply:
Also for the case where we for some reason use the "old" migration way